### DISCUSSION TITLE
Documentation on saving storageblocks on my own db

### DISCUSSION THREAD

**Discussion by WesleyYue** [liveblocks/liveblocks#626] [Help] [answered]
I see the landing page mentions the ability to flush stoageblock data to our own db, but I couldn't find any reference to this in the docs. Where can I read more about this feature?

**Comment by FlowFlorent** [comment, marked as answer]
Hello @WesleyYue !

You can read more about obtaining the storage data of a room on our API docs:
- [Getting storage](https://liveblocks.io/docs/api-reference/rest-api-endpoints#get-rooms-roomId-storage) 
- [Deleting storage](https://liveblocks.io/docs/api-reference/rest-api-endpoints#delete-rooms-roomId-storage) 

Would also be happy to hear more about your use case.

Let me know if this is what you're looking for or if you have additional questions

Cheers

**Reply by WesleyYue** [reply]
Gotcha, so we should basically run a cron job to flush data from the liveblocks regularly?

Btw, I saw the new pricing changes from connections to MAUs which I think makes a lot more sense!

**Reply by FlowFlorent** [reply]
Thank you! We think so too, it'll make for a much more predictable pricing for customers :)

Yes that would be one way of doing it, but, I have good news for you. Your use case would be a perfect fit for our webhooks functionality. We'd like to extend an invitation to you to **join our private beta!** 

Webhooks are an incredibly powerful tool that can be used to send data based on certain Liveblocks events, such as "User Entered Room", "User Left Room" and "Storage Updated". 

If so, I will send you a comprehensive guide on how to implement our webhooks, as well as adding you to a private Discord channel. In this channel, all of our beta testers are able to come together, discuss updates, ask questions, and provide valuable feedback. 

Your input is incredibly important to us, so please feel free to be as open and honest as possible. Let me know if that works for you!

If you're interested, please provide your liveblocks account and discord username so that we can give you access.

Feel free to send this information to support@liveblocks.io or by DM on discord to `@Florent#6395` if you're more comfortable with that.

Cheers
